.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+src/tbbmalloc @ldorau @lplewa @kfilipek
```

Comment:
I see the following error message on both lines 6-7
![image](https://github.com/oneapi-src/oneTBB/assets/19608622/87312c3d-c242-448b-b89d-e6fe75687218)

Reply:
I suppose @ldorau does not exist ;-)

Reply:
> I suppose @ldorau does not exist ;-)

Maybe you don't have write permissions?

Reply:
It seems so, how can I verify that?